utils/reconciler_test.go

```diff
@@ -76,7 +76,7 @@ func TestCreateOrUpdate(t *testing.T) {
 	r := NewReconcilerBase(rcl, cl, s, &rest.Config{}, record.NewFakeRecorder(10))
 
 	err := r.CreateOrUpdate(serviceAccount, runtimecomponent, func() error {
-		CustomizeServiceAccount(serviceAccount, runtimecomponent)
+		CustomizeServiceAccount(serviceAccount, runtimecomponent, r.GetClient())
```

Comment:
should it be return CustomizeServiceAccount(....) ? or it doesn't matter for this test?

Reply:
The test is testing the CreateOrUpdate function, so I don't think the the return value from CustomizeServiceAccount matters (the test could call any function at that point)
That said, I'm not sure the test is a particularly good one as it doesn't seem to verify the behaviour of CreateOrUpdate very much. But if that needs changing, it should probably have a separate issue.